src/strategies/ocean-dao-brightid/index.ts

```diff
@@ -87,6 +103,16 @@ export async function strategy(
     }, {});
     // { address: '0x...55', score: 1.0 }
 
+    // sum delegations
+    addresses.forEach((address) => {
+      if (delegations[address]) {
+        delegations[address].forEach((delegator: string) => {
+          scores[address] += scores[delegator] ?? 0; // add delegator score
+          scores[delegator] = 0; // set delegator score to 0
+        });
+      }
+    });
```

Comment:
Hi! Yes, it's won't work for us because it may cause users to get a double boost from brightid.

Example:
Alice and Bob are verified on Brightid and both have 100 tokens thus their voting power is 400 (boost is 4x).
When we use the delegation strategy: Alice delegates to Bob; Bob has (400+100)*4=2000 voting power.
What we want: Alice delegates to Bob; Bob has (100+100)*4=800 voting power.

Please let me know if I'm missing something.

Reply:
@trizin delegation strategy works same, this is how delegation works:

> This is how delegation works:
Delegation strategy is used get delegated voting power of a address (we can use any strategies to calculate that voting power), but if delegator votes on the proposal, the other person won't get his voting power
For example A have 10 tokens, B got 10
A delegates their voting power to B from https://snapshot.org/#/delegate, there are two cases to calculate Voting power:
> - If Space is using delegation strategy (with erc20-balance-of passed into params)
  B's voting power will be 10 (A delegated voting power + if any other address's delegated voting power)
  A can't vote in this case because no one delegate to him
> - If Space is using delegation strategy (with erc20-balance-of passed into params) + erc20-balance-of strategy
   B's voting power will be 20 (His own tokens +  A delegated voting power + if any other address's delegated voting power)
   A can vote now because they got 10 tokens and If they vote, B's voting power will be reduced to 10 (His own tokens + if any other address's delegated voting power) - He won't have access to A's voting power.

In your case it won't be doubling the voting power i think, it will boost once in delegation strategy if delegator doesn't vote and once in original strategy if the delegator voted

Reply:
I was talking about the boost here:
https://github.com/snapshot-labs/snapshot-strategies/blob/3ae7a4bf68f14b77c62ba3e2b716ef622760bffb/src/strategies/ocean-dao-brightid/index.ts#L126-L128
Let's say options.brightIdMultiplier=4
If the delegator and the delegated address are verified on BrightId, the total boost will be 16x instead of 4x.

Reply:
I don't think so @trizin boost will be applied only once, either inside the delegation strategy or the original strategy, Can you try passing your strategy into the delegation strategy?